src/main/java/com/group/CSStudentJobSearchApp/service/JobService.java

```diff
@@ -0,0 +1,30 @@
+package com.group.CSStudentJobSearchApp.service;
+
+import com.group.CSStudentJobSearchApp.dao.JobRepository;
+import com.group.CSStudentJobSearchApp.model.Job;
+import com.group.CSStudentJobSearchApp.model.Location;
+import com.group.CSStudentJobSearchApp.model.Skill;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+import java.util.UUID;
+
+@Service
+public class JobService {
+    private final JobRepository jobRepository;
+
+    @Autowired
+    public JobService(JobRepository jobRepository) {
+        this.jobRepository = jobRepository;
+    }
+
+    public List<Job> getJobs() {
+        return jobRepository.findAll();
+    }
+
+    public void addNewJob(Job job) {
+        System.out.println("post is working");
+        System.out.println(job);
```

Comment:
Just to double check, this is only sending a request to the controller, right? It's not actually adding anything to the database yet for POST requests?

Reply:
not yet, I am waiting Andrea to give me some real data